src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -79,6 +80,20 @@ function attachVisibilityDropdownEvent() {
     });
 }
 
+function checkAndMarkDestructiveChange(selectedOption, $containingForm) {
+    if (selectedOption === 'Custom visibility options...') {
+        return;
```

Comment:
@wkurniawan07 You are correct. When clicking on the "custom visibility option", there is no change to the existing state/selection of the checkboxes because all that happens is that the checkbox div is set to visible.